Topics/Tech_Stacks/DigitalOcean_Web_App.md

```diff
@@ -0,0 +1,129 @@
+# Deploying a Web Application on DigitalOcean
+
+## Introduction
+This tutorial will guide you step-by-step through the process of deploying a web application using DigitalOcean's hosting platform. DigitalOcean, a cloud service provider, provides a straightforward and user-friendly environment for hosting applications, making it an excellent choice for developers to use.
+
+## Prerequisites
+Before you begin, ensure you have the following:
+
+1. **DigitalOcean Account**: Sign up for a DigitalOcean account if you don't have one already. (https://cloud.digitalocean.com/registrations/new)
+
+2. **Web Application Code**: Have your web application code ready for production. This could be a Node.js, Python, Ruby on Rails, or any other type of application.
+
+Now that you have these two prerequisites, you're ready to launch your web app into the world! Follow these steps:
+
+### Step 1: Create a Droplet
+In digital ocean, a "Droplet" refers to a virtual private server (VPS) that you can deploy and manage on DigitalOcean's cloud infrastructure. It's essentially a scalable compute platform with add-on storage, where you can run applications, websites, and other services.
```

Comment:
Is digital ocean a separate term to the Digital Ocean company?
and also computing instead of compute